index.js

```diff
@@ -162,6 +164,87 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.slice(8);
+
+  axios
+    .get(
+      "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=" + fuzzySearchKey
+    )
+    .then(function (cardSearchResults) {
+      console.log(
+        "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=" + fuzzySearchKey
```

Comment:
Any duplication is a code smell -- in this case the solution is easy: we can just store this string as a `const` and reuse it